### PR TITLE
update ABAQUS easyblock for recent versions, incl. support for installing hotfixes

### PR DESCRIPTION
**Pull request by boegel**
(still WIP because of the `FIXME` that needs to be taken care of, once I get my hands on the required installation file)

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 Early thoughts on this?

File: easybuild/easyblocks/a/abaqus.py [hunk 1]
```diff
@@ -69,31 +77,100 @@ def configure_step(self):
...
+                # first install Part_3DEXP_SimulationServices hotfix(es)
+                hotfix_dir = os.path.join(self.builddir, 'Part_3DEXP_SimulationServices.Linux64', '1', 'Software')
+                change_dir(hotfix_dir)
+
+                # SIMULIA_ComputeServices part
+                subdirs = glob.glob('HF_SIMULIA_ComputeServices.HF*.Linux64')
+                if len(subdirs) == 1:
+                    subdir = subdirs[0]
+                else:
+                    raise EasyBuildError("Failed to find expected subdir for hotfix: %s", subdirs)
```
Comment (wpoely86) [inline easybuild/easyblocks/a/abaqus.py]: this is fatal? Not sure I understand?

**Comment by wpoely86** [comment]
looks fine

File: easybuild/easyblocks/a/abaqus.py (hunk 1 above)
Comment (boegel) [inline easybuild/easyblocks/a/abaqus.py]: Yes, if the expected subdirectory is not found, something is very wrong...

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**